src/server/server_family.cc

```diff
@@ -721,6 +733,26 @@ void ServerFamily::SnapshotScheduling(const SnapshotSpec& spec) {
   }
 }
 
+void ServerFamily::SnapshotScheduling(const cron::cronexpr& cron_expr) {
```

Comment:
As mentioned above, we can probably consolidate this function and the other overload